contrib_bots/lib/john.py

```diff
@@ -0,0 +1,131 @@
+# pip install chatterbot is necessary
+import os
+import json
+import time
+import threading
+from random import choice
+from chatterbot import ChatBot
+from chatterbot.trainers import ChatterBotCorpusTrainer, ListTrainer
+jokes_path = os.path.expanduser('lib/John/jokes.json')
+database_path = os.path.expanduser('~/JohnBot/database.db')
+# Create a new instance of a ChatBot
+bot = ChatBot("John",
+              storage_adapter="chatterbot.storage.JsonFileStorageAdapter",
+              logic_adapters=
+              [
+                  "chatterbot.logic.MathematicalEvaluation",
+                  {
+                      "import_path": "chatterbot.logic.BestMatch",
+                      "response_selection_method": "chatterbot.response_selection.get_random_response",
+                      "statement_comparison_function": "chatterbot.comparisons.levenshtein_distance"
+                  }],
+              output_adapter="chatterbot.output.OutputFormatAdapter",
+              output_format='text',
+              database=database_path,
+              silence_performance_warning="True")
+
+bot.set_trainer(ListTrainer)
+
+bot.train([
+    "I want to contribute",
+    """Contributors are more than welcomed! Please read
+    https://github.com/zulip/zulip#how-to-get-involved-with-contributing-to-zulip
+    to learn how to contribute.""",
+])
+
+bot.train([
+    "What is Zulip?",
+    """Zulip is a powerful, open source group chat application. Written in Python
+    and using the Django framework, Zulip supports both private messaging and group
+    chats via conversation streams. You can learn more about the product and its
+    features at https://www.zulip.org.""",
+])
+
+bot.train([
+    "I would like to request a remote dev instance",
+    """Greetings! You should receive a response from one of our mentors soon.
+    In the meantime, why don't you learn more about running Zulip on a development
+    environment? https://zulip.readthedocs.io/en/latest/using-dev-environment.html""",
+])
+
+bot.train([
+    "Joke!",
+    "Only if you ask nicely!",
+])
+
+bot.train([
+    "What is your name?",
+    "I am John, my job is to assist you with Zulip.",
+])
+
+bot.train([
+    "What can you do?",
+    "I can provide useful information and jokes if you follow etiquette.",
+])
+
+with open(jokes_path) as data_file:
+    for joke in json.load(data_file):
+        bot.train([
+            "Please can you tell me a joke?",
+            list(joke.values())[0],
```

Comment:
Are you trying to get the value of the dictionary here? I think just `joke['joke']` should work.